core/server/config/overrides.json

```diff
@@ -5,6 +5,7 @@
         "clientAssets": "core/built/assets",
         "helperTemplates": "core/server/helpers/tpl/",
         "adminViews": "core/server/views/",
+        "defaultViews": "core/server/views/",
```

Comment:
Yes, they are not the same thing - adminViews are for the admin panel, defaultViews are fallbacks for themes. They live in the same place for noooow. Admin views will change here: https://github.com/TryGhost/Ghost/pull/8142/files#diff-4a9ff835bae8d805a7741e6356728bcfR5